standard-controller/src/main/java/io/enmasse/controller/standard/AddressController.java

```diff
@@ -154,6 +158,41 @@ public void onUpdate(Set<Address> addressSet) throws Exception {
 
     }
 
+    private Set<Address> filterStatusQuo(Set<Address> addressSet) {
+        Set<Address> statusQuo = new HashSet<>();
+        for (Address address : addressSet) {
+            switch (address.getStatus().getPhase()) {
+                case Failed:
+                case Terminating:
+                case Active:
+                case Configuring:
+                    if (address.getStatus().getActivePlan() == null || address.getStatus().getActivePlan().equals(address.getPlan())) {
```

Comment:
can you explain me please in which cases will be status.activePlan == address.getPlan() ?

Reply:
This will be the case once the new plan has been 'activated'. i.e. quota check has passed (in case of requiring more resources), and it has started to scaled down or up.